bin/release_manager.py

```diff
@@ -39,84 +44,96 @@ def get_rc_release(version):
 
     raise exception_from_error("Unknown error while looking RC release: ", response)
 
+
 def create_release(version, commit_sha):
     tag = rc_tag_name(version)
 
-    params = {
```

Comment:
I think this style is better than your changed style.

Reply:
For readability, I think it`s better not to expand the code here.

Reply:
I might need to look into the configuration of black.